insights/specs/default.py

```diff
@@ -335,29 +350,16 @@ def du_dirs_list(broker):
     jbcs_httpd24_httpd_error_log = simple_file("/opt/rh/jbcs-httpd24/root/etc/httpd/logs/error_log")
     virt_uuid_facts = simple_file("/etc/rhsm/facts/virt_uuid.facts")
 
-    @datasource(ps_auxww)
+    @datasource(PsAuxcww)
     def httpd_cmd(broker):
         """
-        Function to search the output of ``ps auxww`` to find all running Apache
+        Function to search the output of ``ps auxcww`` to find all running Apache
         webserver processes and extract the binary path.
 
         Returns:
             list: List of the binary paths to each running process
         """
-        ps = broker[DefaultSpecs.ps_auxww].content
-        ps_httpds = set()
-        for p in ps:
-            p_splits = p.split(None, 10)
-            if len(p_splits) >= 11:
-                cmd = p_splits[10].split()[0]
-                # Should compatible with RHEL6
-                # e.g. /usr/sbin/httpd, /usr/sbin/httpd.worker and /usr/sbin/httpd.event
-                #      and SCL's httpd24-httpd
-                if os.path.basename(cmd).startswith('httpd'):
-                    ps_httpds.add(cmd)
-        # Running multiple httpd instances on RHEL is supported
-        # https://access.redhat.com/solutions/21680
-        return list(ps_httpds)
+        return _get_running_commands(broker, 'httpd')
```

Comment:
Another reason to use `list(set())` in `_get_running_command` since this originally returned a `list`.